waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -508,28 +513,33 @@ proc getNumStreams*(pm: PeerManager, protocol: string): (int, int) =
               numStreamsOut += 1
   return (numStreamsIn, numStreamsOut)
 
+proc pruneInRelayConns(pm: PeerManager, amount: int) {.async.} =
+  let (inRelayPeers, outRelayPeers) = pm.connectedPeers(WakuRelayCodec)
+  let connsToPrune = min(amount, inRelayPeers.len)
```

Comment:
```suggestion
proc pruneInRelayConns(pm: PeerManager, connsToPrune: int) {.async.} =
```
As per how this proc is used, the `min(amount, inRelayPeers.len)` will always return `amount`.

Reply:
yup, as its used yep but you never know hoy a function will be used. good sanity check to avoid accessing out of bound elements.

Reply:
Cool, makes sense!

Reply:
Would it make sense to pass list of peerIds to prune here though? Instead of `amount` - that way we could avoid counting/getting the peers again (you already do it before calling this function) and it would allow for potentially "smarter" ways of choosing the peers to prune - this just goes from first peer, but by passing an array of peerIds to prune the caller could come up with a different way how to choose the peers to prune